gulpfile.js

```diff
@@ -330,7 +330,7 @@ gulp.task('copy', 'Copy site files (polyfills, templates, etc.) to dist/', funct
       '*',
       'app/manifest.json',
       'app/service-worker.js',
-      '!{README.md,package.json,gulpfile.js,test_runner.py}',
```

Comment:
D'oh. Never mind—test_runner.py was _removed_ from the  exclude list, which is correct... For some reason, this change showed up as the only relevant change in the merge commit (thx github) so I thought something had gotten reversed by accident. Plz ignore.